README.md

```diff
@@ -24,6 +24,14 @@ Simply running a container with the image will launch a Perl 6 REPL:
 You can also provide perl6 command line switches to `docker run`:
 
     $ docker run -it rakudo-star perl6 -e 'say "Hello!"'
+	
+or run your own files using the container's interpreter
+
+	$ docker run -t -v `pwd`:/home/perl6/app rakudo-star /home/perl6/app/ls.p6
+
+where we mal the local file ls.p6 (in this directory) to the remote
```

Comment:
Volume is not really needed. It's just a convenient place to put scripts. You can create new volumes with -v all the time.